client/lib/setup/index.coffee

```diff
@@ -49,6 +54,10 @@ module.exports = (app) ->
   app.use bucketAssets()
   setupAuth app
   app.use locals
+
+  if SENTRY_PRIVATE_DSN
+    app.use RavenServer.errorHandler()
```

Comment:
@damassi correct me if I'm wrong but I think this does happen before all other error handling. The main error handler is at the end of the file. Pretty sure it should be moved to after we mount the sub-apps tho. I'll do that now!